src/Controller/EnMarche/ProcurationController.php

```diff
@@ -37,11 +37,13 @@ public function landingAction(Request $request): Response
      */
     public function indexAction(Request $request): Response
     {
-        $this->getProcurationFlow()->reset();
+        $procurationSession = $this->get(ProcurationRequestSession::class);
```

Comment:
You can use Action injection instead

Reply:
Did not know it was set up, done. Thanks again!